test/tests/functional/pbs_hooksmoketest.py

```diff
@@ -49,32 +45,32 @@ def get_file_path(self, hook_name):
         self.logger.info("As expected dir '%s' present" % server_hooks_dir)
         hk_file = hook_name + ".HK"
         hk_file_location = os.path.join(server_hooks_dir, hk_file)
-        return server_hooks_dir, hk_file, hk_file_location
-
-    def check_hk_file(self, existence):
+        self.logger.info("Check existence of .HK file")
+        count = 2
         if existence:
-            msg = "As expected file '%s' is present" % self.hk_file
-            msg += " in '%s' directory" % self.server_hooks_dir
-            _msg = "File '%s' is not present" % self.hk_file
-            _msg += " in '%s' directory" % self.server_hooks_dir
+            count = 10
+            msg = "As expected file '%s' is present" % hk_file
+            msg += " in '%s' directory" % server_hooks_dir
+            _msg = "File '%s' is not present" % hk_file
+            _msg += " in '%s' directory" % server_hooks_dir
         else:
-            msg = "As expected file '%s' is not present" % self.hk_file
-            msg += " in '%s' directory" % self.server_hooks_dir
-            _msg = "File '%s' is present" % self.hk_file
-            _msg += " in '%s' directory" % self.server_hooks_dir
+            msg = "As expected file '%s' is not present" % hk_file
+            msg += " in '%s' directory" % server_hooks_dir
+            _msg = "File '%s' is present" % hk_file
+            _msg += " in '%s' directory" % server_hooks_dir
 
         # sleeping for some time as generation of *.HK file takes time
-        count = 5
         while True:
             rc = self.du.isfile(hostname=self.server.hostname,
-                                path=self.hk_file_location,
+                                path=hk_file_location,
                                 sudo=True)
             count = count - 1
             if rc or count == 0:
                 break
             time.sleep(1)
         self.assertEqual(rc, existence, _msg)
         self.logger.info(msg)
+        return server_hooks_dir, hk_file, hk_file_location
```

Comment:
the function does not need to return the paths anymore, the caller is not using them.